source/reference/kubectl-minio-plugin/kubectl-minio-tenant-info.rst

```diff
@@ -27,6 +27,7 @@ Displays information on a MinIO Tenant, including but not limited to:
 - The total capacity of the Tenant
 - The version of MinIO server and MinIO Console running on the Tenant
 - The configuration of each Pool in the Tenant.
+- The root user credentials for the Tenant.
```

Comment:
Could also add an `Example` section. The actual example would duplicate the one already given, but there's more room for sample output. 

If that seems useful. The output is straightforward however.

Reply:
There's no harm in it if you want to extend the scope slightly.